advanced/src/components/SuperContainer.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import {
+  NavLink,
+  Link,
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  Redirect
+} from 'react-router-dom'
+import FeedPage from './FeedPage'
+import DraftsPage from './DraftsPage'
+import CreatePage from './CreatePage'
+import DetailPage from './DetailPage'
+import LoginPage from './LoginPage'
+import SignupPage from './SignupPage'
+import PageNotFound from './PageNotFound'
+import LogoutPage from './LogoutPage'
+import { ApolloProvider } from 'react-apollo'
+import { AUTH_TOKEN } from '../constant'
+
+
+const ProtectedRoute = ({component: Component, token, ...rest}) => {
+    return  token ?
+    (<Route {...rest} render={matchProps => (<Component {...matchProps} />)} />)
+    :(<Redirect to="/login" />)
+};
+
+class SuperContainer extends React.Component {
+
+  constructor(props) {
+    super(props)
+
+    this.refreshFn = this.refreshFn.bind(this)
+
+    this.state = {
+      token : props.token
+    }
+  }
+
+  refreshFn(data= {}) {
+    this.setState({
+      token : data.token
+    })
+  }
+
+  render() {
+    return (<ApolloProvider client={this.props.client}>
+      <Router>
+        <React.Fragment>
+          <nav className="pa3 pa4-ns">
+            <Link
+              className="link dim black b f6 f5-ns dib mr3"
+              to="/"
+              title="Feed"
+            >
+              Blog
+            </Link>
+            <NavLink
+              className="link dim f6 f5-ns dib mr3 black"
+              activeClassName="gray"
+              exact={true}
+              to="/"
+              title="Feed"
+            >
+              Feed
```

Comment:
Looks like you have two links routing to `/`

Reply:
Should I remove it ? I think the basic also has the same thing and that is why I kept it similar to basic boilerplate. Let me know if we plan to remove Blog? We can remove it from both basic & advanced both :)

Reply:
Yeah, I would. I haven't tracked these boilerplates so I'm not sure the reason why it was implemented in basic. @DevanB would probably be able to answer. 😄

Reply:
I guess we can remove it but I will wait untill @DevanB or @nikolasburk approves it ;)